velox/docs/functions/uuid.rst

```diff
@@ -0,0 +1,7 @@
+=============================
+UUID Functions
+=============================
+
+.. function:: uuid() -> uuid
+
+    Returns a pseudo randomly generated UUID as a single character string
```

Comment:
> as a single character string

This doesn't seem to be correct. Would you double check?

Also, Presto specifically says that it generated UUID type 4. What kind of UUID is generated here?

```
Returns a pseudo randomly generated [UUID](https://prestodb.io/docs/current/language/types.html#uuid-type) (type 4).
```